markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -69,6 +73,24 @@ library PerpsAccount {
         }
     }
 
+    function canOpenNewPosition(Data storage self) internal view {
```

Comment:
would rename this to `validateMaxPositions`.  `can` implies it will return a boolean.